_episodes/03-create.md

```diff
@@ -447,18 +455,30 @@ quotes.txt
 >
 > ~~~
 > $ ls -F
+> ~~~
+> ~~~
 >  analyzed/ raw/
+> ~~~
+> {: .output}
+> ~~~
 > $ ls -F analyzed
+> ~~~
+> {: .language-bash}
+> ~~~
 > fructose.dat glucose.dat maltose.dat sucrose.dat
+> ~~~
+> {: .output}
+> ~~~
 > $ cd raw/
 > ~~~
 > {: .language-bash}
 >
-> Fill in the blanks to move these files to the current folder
+> Replace `placeholder1`, `placeholder2` and `placeholder3`PLACEHOLDER2 in the blanks
```

Comment:
I think PLACEHOLDER2 is a typo?
Also I don't think the rewording makes sense. There are now no blanks. I think the question was fine, but if it's going to be changed, why not use something more descriptive than 'placeholder1'? Maybe 'directory1'?

Reply:
a4978c30d37db478001dbfafd94c7bdd2837811d reverts to the use of underline.